apps/i18n/common/en_us.json

```diff
@@ -1966,7 +1967,7 @@
   "watchVideo": "Watch the Video",
   "weekLabel": "Week {number}",
   "weeklyLessonLayout": "Weekly Lesson Layout",
-  "weeklyLessonLayoutDescription": "A weekly chart of a suggested schedule based on {minutes} instructional minutes per week.",
+  "weeklyLessonLayoutDescription": "Instructional minutes per week",
```

Comment:
Just an FYI once a string goes in to the system and gets translated if you change the string it will only updated for the english version right away and other languages will still use the old version until a new translation is given. So in general its better to just create a new entry and delete the old one if that is what you are doing.

Reply:
Since we haven't launch this feature you could wait and merge all the PRs together so then it wouldn't really be an issue. Or you could update the earlier PR to just use this language to start even though its not perfect for that PR. Or just pick a new key string here. Whatever works for you